staging/src/k8s.io/apiserver/pkg/authorization/authorizerfactory/delegating.go

```diff
@@ -36,12 +37,28 @@ type DelegatingAuthorizerConfig struct {
 	// DenyCacheTTL is the length of time that an unsuccessful authorization response will be cached.
 	// You generally want more responsive, "deny, try again" flows.
 	DenyCacheTTL time.Duration
+
+	// SkippedPaths are HTTP paths which are excluded from authentication. They can be plain
+	// paths or end in * in which case prefix-match is applied. A leading / is optional.
+	SkippedPaths []string
 }
 
 func (c DelegatingAuthorizerConfig) New() (authorizer.Authorizer, error) {
-	return webhook.NewFromInterface(
+	var authorizers []authorizer.Authorizer
+
+	if len(c.SkippedPaths) > 0 {
+		authorizers = append(authorizers, path.NewAuthorizer(c.SkippedPaths))
```

Comment:
I would expect the union authorizer to be constructed in the component and the delegate authorizer factory to remain unchanged.

Reply:
Use case is delegated authz for endpoints on components like controller manager, etc, while letting healthz punch through so that control plane being down doesn't block ability to run liveness probes

Reply:
ok, makes sense. I think you were responding to my other comment.

Reply:
Or do you mean that this should be inextricably bundled with the delegated auth config because of that?

Reply:
Yes, I agree.

done